src/cloud/components/CardinalityLimitAlertContent.tsx

```diff
@@ -115,30 +116,37 @@ export const UpgradeContent: FC<UpgradeProps> = ({
   )
 }
 
-export const RateLimitAlertContent: FC<Props> = ({className, location}) => {
+export const CardinalityLimitAlertContent: FC<Props> = ({
+  className,
```

Comment:
This component only does Cardinality alerts, so let's avoid confusion with the other more generic 'Rate Limit Alert' components.